diff --git a/utils/notify/notification.go b/utils/notify/notification.go
index 0b3421c..f4a018f 100644
--- a/utils/notify/notification.go
+++ b/utils/notify/notification.go
@@ -126,7 +126,9 @@ func Notify(content NotifyContent) {
 		AppName: types.AppName,
 	})
 
-	if runtime.GOOS != "darwin" {
+	if os.Getenv("container") == "flatpak" {
+		content.Icon = "/app/share/icons/hicolor/512x512/apps/io.github.go_musicfox.go-musicfox.png"
+	} else if runtime.GOOS != "darwin" {
 		localDir := app.DataRootDir()
 		content.Icon = filepath.Join(localDir, configs.ConfigRegistry.Main.NotifyIcon)
 		if _, err := os.Stat(content.Icon); os.IsNotExist(err) {
